example/lambdas/ftpPopulateTestLambda/package.json

```diff
@@ -19,21 +19,11 @@
     "access": "private"
```

Comment:
This isn't really relevant to this PR but I am seeing this locally in my editor
```
Value is not accepted. Valid values: "public", "restricted"
```

Reply:
Yeah, that's not great.   We should update this on the main branch.